src/original/PvH_KB_NKI/DWI_functions_standalone.py

```diff
@@ -0,0 +1,239 @@
+f# -*- coding: utf-8 -*-
```

Comment:
This `f` seems to be causing the test parser to choke.

Reply:
Sorry for the delay. I removed the `f` that was causing this error.

Reply:
Perhaps just change the path to the current directory so the function runs. Maybe add a comment with the YOUR_PATH so people know. I'll also try to see if there's a way to avoid these failures in the future.